package.json

```diff
@@ -76,7 +76,7 @@
     "eslint-plugin-import": "2.x",
     "linklocal": "^2.8.2",
     "mocha": "^3.5.3",
-    "nyc": "^11.9.0",
+    "nyc": "^15.0.0",
```

Comment:
I'd prefer not to upgrade this; i've asked on https://github.com/istanbuljs/nyc/issues/1205 about a workaround.

Reply:
Another option would be to decouple testing and coverage collection and either

- update to `nyc@15` and stop collecting coverage on node@<8
- stop collecting coverage on windows